### PR TITLE
Combine all the samples into one project to share the configuration

### PR DESCRIPTION
**Pull request by joyl1216**
Merge all the existing samples into one project, so they can share the same configuration and dependencies. The pipeline is finally satisfied with the latest Kotlin and gradle version.

### CODE REVIEW

File: ComposeSamples/ListDetail/src/main/java/com/microsoft/device/display/samples/listdetail/DetailView.kt [hunk 1]
```diff
@@ -0,0 +1,183 @@
+package com.microsoft.device.display.samples.listdetail
```
Comment (CesarValiente) [inline ComposeSamples/ListDetail/src/main/java/com/microsoft/device/display/samples/listdetail/DetailView.kt:1]: Missed OSS header here

**Review by CesarValiente — CHANGES_REQUESTED** [review]
LGTM! Just one but important and tedious thing to do, please Joy review all files and add the OSS (MSFT MIT) mit header to all files before merging this PR.

Thanks, and wonderful work!

**Comment by joyl1216** [comment]
> LGTM! Just one but important and tedious thing to do, please Joy review all files and add the OSS (MSFT MIT) mit header to all files before merging this PR.
> 
> Thanks, and wonderful work!

Sure, thanks! Updated.

**Review by CesarValiente — APPROVED** [review]
LGTM then!! Thanks Joy!

**Pull request closed without merging**